pkgs/applications/networking/browsers/mullvad-browser/default.nix

```diff
@@ -0,0 +1,224 @@
+{ lib, stdenv
+, fetchurl
+, makeDesktopItem
+, makeWrapper
+
+# Common run-time dependencies
+, zlib
+
+# libxul run-time dependencies
+, atk
+, cairo
+, dbus
+, dbus-glib
+, fontconfig
+, freetype
+, gdk-pixbuf
+, glib
+, gtk3
+, libxcb
+, libX11
+, libXext
+, libXrender
+, libXt
+, libXtst
+, pango
+
+, libnotifySupport ? stdenv.isLinux
+, libnotify ? libnotifySupport
+
+, audioSupport ? mediaSupport
+, pulseaudioSupport ? mediaSupport
+, libpulseaudio
+, apulse
+, alsa-lib
+
+# Media support (implies audio support)
+, mediaSupport ? true
+, ffmpeg
```

Comment:
Should we assert this?

Reply:
I don't think it's necessary. If the user wants to disable `audioSupport` & `pulseaudioSupport` but keep `mediaSupport` enabled that might actually be useful when a new media-related option is added later.